src/main/java/org/team1540/robot2019/OI.java

```diff
@@ -122,6 +130,10 @@ public static void initButtons() {
         climbLevel2Button.whenPressed(new SimpleConditionalCommand(climbingSafety::get, new ClimbLevelTwo()));
         climberCylinderUp.whenPressed(new SimpleCommand("Raise Cylinder", Robot.climber::cylinderUp, Robot.climber));
 
+        Command alignCommand = new PurePursuitThenPointToVisionTarget();
+        autoAlignButton.whenPressed(alignCommand);
+        autoAlignCancelButton.whenPressed(new SimpleCommand("Cancel Auto-lineup", alignCommand::cancel));
```

Comment:
It turns out that there's actually a `cancelWhenPressed()` method; can we switch to that?

Reply:
Done.